uPortal-security/uPortal-security-permissions/src/main/java/org/apereo/portal/security/provider/AuthorizationImpl.java

```diff
@@ -73,8 +75,18 @@
 @Service("authorizationService")
 public class AuthorizationImpl implements IAuthorizationService {
 
+    /**
+     * Period during which this service will not complain (in the logs) about the same item of
+     * missing data, currently 5 minutes. Without this throttle, the logs would quickly fill with
+     * the same message when there is missing data.
+     */
+    private static final long MISSING_DATA_LOG_PERIOD_MILLIS =
```

Comment:
https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/TimeUnit.html

```java
TimeUnit.Minutes.toMillis(5L);
```

Reply:
Needs to be...

```
TimeUnit.MINUTES.toMillis(5L);
```

but not a bad suggestion.